services/storage/store.go

```diff
@@ -385,53 +462,159 @@ func (s *Store) TagValues(ctx context.Context, req *datatypes.TagValuesRequest)
 	return cursors.NewStringSliceIterator(names), nil
 }
 
-type MeasurementNamesRequest struct {
-	MeasurementsSource *types.Any
-	Predicate          *datatypes.Predicate
+func (s *Store) MeasurementNames(ctx context.Context, mqAttrs *metaqueryAttributes) (cursors.StringIterator, error) {
```

Comment:
Changing this will also require changing the Enterprise RPC requests

Reply:
Good point... should `metaqueryAttributes` be `MetaqueryAttributes` for it to be accessible from plutonium?

Reply:
Hrm, it looks like enterprise contains a copy-paste of the old code, with no RPC for the Flux meta queries. Is that expected?